Sources/NIOChatClient/main.swift

```diff
@@ -44,7 +44,7 @@ private final class ChatHandler: ChannelInboundHandler {
 let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
 let bootstrap = ClientBootstrap(group: group)
     // Enable SO_REUSEADDR.
-    .channelOption(ChannelOptions.socket(SocketOptionLevel(SOL_SOCKET), SO_REUSEADDR), value: 1)
+    .channelOption(ChannelOptions.Types.SocketOption.allowLocalAddressReuse, value: 1)
```

Comment:
we shouldn't expose the `Types` thing in actual code unless really necessary. I think we should have something like

    ChannelOptions.allowLocalAddressReuse, value: true

Reply:
The `Types` is just an artefact of how this is implemented under the hood but the user shouldn't ever see this (unless they're defining their own channel options).